.github/workflows/pull-request-test.yml

```diff
@@ -162,37 +172,28 @@ jobs:
         pip install yq
     - name: Test the PR
       env:
-        RENKU_BOT_DEV_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
+        KUBECONFIG: ${{ github.workspace }}/renkubot-kube.config
+        RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_KUBECONFIG }}
+        RELEASE: ci-${{ github.event.number }}-renku
       run: |
-        export PR_NUMBER=${{ github.event.number }}
-        echo $PR_NUMBER
-        export RENKU_RELEASE="ci-${PR_NUMBER}-renku"
-
-        RENKU_PYTHON_VERSION=v$(cat ./charts/renku/requirements.yaml | yq -r '.dependencies[] | select(.name == "renku-core") | .version')
-        RENKU_PYTHON_COMMIT=$(echo $RENKU_PYTHON_VERSION | grep -o -P '(?<=-)[0-9a-fA-F]+') || true
-        if [ $RENKU_PYTHON_COMMIT ]; then export RENKU_PYTHON_VERSION=$RENKU_PYTHON_COMMIT; fi
-        echo "Passing renku-python version $RENKU_PYTHON_VERSION"
-
-        cd acceptance-tests
-        COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build --build-arg renku_python_ref=${RENKU_PYTHON_VERSION} sbt
-        docker-compose run -e RENKU_TEST_URL=https://${RENKU_RELEASE}.dev.renku.ch \
-                           -e RENKU_TEST_FULL_NAME="Renku Bot" \
-                           -e RENKU_TEST_EMAIL="renku@datascience.ch" \
-                           -e RENKU_TEST_REGISTER="1" \
-                           -e RENKU_TEST_USERNAME="renku-test" \
-                           -e RENKU_TEST_PASSWORD="$RENKU_BOT_DEV_PASSWORD" \
-                           -e RENKU_TEST_ANON_AVAILABLE="true" sbt
-    - name: Prepare artifact for packaging on failure
+        echo "$RENKUBOT_KUBECONFIG" > renkubot-kube.config
+        helm test $RELEASE --namespace $RELEASE --timeout 40m --logs
+    - name: Download artifact for packaging on failure
+      env:
+        OS_DEV_ACCESS: ${{ secrets.OS_DEV_ACCESS }}
+        OS_DEV_SECRET: ${{ secrets.OS_DEV_SECRET }}
       if: failure()
```

Comment:
Does the above `helm upgrade --install` command wait for the tests to pass/fail before returning and return a corresponding return value?

Reply:
Perhaps you mean the `helm test` command? In that case yes, it wait for the tests to complete (e.g. it really just waits for the pods to terminate) and return the corresponding return code.

Reply:
Sorry yeah, of course the tests are running on `helm test` and not automatically after the install...